internal/config/cgmgr/cgmgr.go

```diff
@@ -79,8 +88,58 @@ func SetCgroupManager(cgroupManager string) (CgroupManager, error) {
 		}
 		return &systemdMgr, nil
 	case cgroupfsCgroupManager:
-		return new(CgroupfsManager), nil
+		return &CgroupfsManager{
+			memoryPath:    cgroupMemoryPathV1,
+			memoryMaxFile: cgroupMemoryMaxFileV1,
+		}, nil
 	default:
 		return nil, errors.Errorf("invalid cgroup manager: %s", cgroupManager)
 	}
 }
+
+func verifyCgroupHasEnoughMemory(cgroupManager, cgroupParent, memorySubsystemPath, memoryMaxFilename string) error {
+	var (
+		slicePath string
+		err       error
+	)
+
+	if cgroupManager == systemdCgroupManager {
+		slicePath, err = systemd.ExpandSlice(cgroupParent)
+		if err != nil {
+			return errors.Wrapf(err, "expanding systemd slice path for %q", cgroupParent)
+		}
+	} else {
+		slicePath = cgroupParent
+	}
```

Comment:
instead of branching here, can we pass in slicePath to verifyCgroupHasEnoughMemory() (ExpandSlice() from systemd, straight up from cgroupParent)? it doesn't seem we need cgroupParent and we'd also not need cgroupManager

Reply:
Yeah. That will make it look better.